docs/source/user/aerodyn/modeling.rst

```diff
@@ -114,4 +114,5 @@ linearization of the full coupled solution. When induction is enabled
 (``WakeMod = 1``), we recommend to base the linearized solution on the
 frozen-wake assumption, by setting ``FrozenWake = TRUE``. The UA
 models are not set up to support linearization, so, UA must be disabled
-during linearization by setting ``AFAeroMod = 1``.
+during linearization by setting ``AFAeroMod = 1``. Linearization is not 
+currently possible when modeling an MHK turbine.
```

Comment:
Is this true?  What is preventing linearization from functioning properly?  Perhaps because of the nacelle and hub buoyancy loads?

Reply:
Yes, this is in the documentation because the hub and nacelle loads are not yet included in the linearization. Would you recommend removing or rephrasing this statement to be more specific?

Reply:
OK, thanks for confirming.  You could add something like, "We intend to enable support for AeroDyn linearization for MHK turbines in an upcoming release."

Reply:
Sounds good, I will make that change.

Reply:
I've added this in the updated doc.